src/composables/useDelegates.ts

```diff
@@ -93,6 +97,7 @@ export function useDelegates(delegationApiUrl: string) {
         (Number(delegate.delegatedVotes) / Number(governanceData.delegatedVotes)) * 100;
 
       return {
+        name: names[delegate.id] || null,
```

Comment:
It should be added in the type as well:
https://github.com/snapshot-labs/sx-ui/blob/5e40f34bbd0931ccef8f22cebf178726e1aa74c2/src/composables/useDelegates.ts#L12-L15